.github/workflows/test.yml

```diff
@@ -43,12 +43,8 @@ jobs:
       run: |
         choco install vcpython27 -f -y
 
-    - name: Get PR labels
-      id: pr-labels
-      uses: joerick/pr-labels-action@v1.0.6
-
     - name: Sample build
-      if: contains(steps.pr-labels.outputs.labels, ' ci-sample-build ')
+      if: "contains(github.event.pull_request.labels.*.name, 'CI: Sample build')"
```

Comment:
We should also decide what we want to do for non pull_request builds. Should we only build samples if requested on pull_requests? It should also be possible via `workflow_dispatch`, what about on master?

Reply:
What if we set up some kind of weekly cron trigger? I had already forgotten there was this label to trigger things :-/ Once a week (or so) doesn't seem to crazy when it comes to using resources?